CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # CHANGELOG for ChefSpec
 
+## 7.0.0 (April, 18, 2017)
+
+- Increased the chef gem dependency from 12.6+ to 12.14+. Since ChefSpec is consumed within ChefDK this actually has a minor impact on end users who generally consume Chefspec via ChefDK, which will bundle both compatible chef and chefspec releases.
```

Comment:
Because it makes me twitch slightly, should be consistent on if the lines end with periods or not :)